src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -73,7 +73,7 @@
  */
 public class Endpoint
     extends AbstractEndpoint implements PotentialPacketHandler,
-        EncodingsManager.EncodingsUpdateListener
+        EncodingsManager.EncodingsUpdateListener, TransceiverEventHandler
```

Comment:
For implementations of interfaces that will never be called by an external user of the class, I think it's nice to do "private" implementations of the interface (e.g. via a private inner class).  It also moves the methods out of the 'top level' of the class which I think is cleaner.